code/modules/overmap/ships/computers/ship.dm

```diff
@@ -60,7 +60,11 @@ somewhere on that shuttle. Subtypes of these can be then used to perform ship ov
 	if(linked)
 		user.reset_view(linked)
 	if(user.client)
-		user.client.view = world.view + extra_view
+		if(istext(user.client.view))
+			var/list/retrieved_view = splittext(user.client.view, "x")
+			user.client.view = "[text2num(retrieved_view[1]) + extra_view]x[text2num(retrieved_view[2]) + extra_view]"
```

Comment:
Probably out of scope since this is a fix for existing code, but extra view should be handled by the client in OnResize() or something. If they user resizes their window while looking through this console they'll lose the extra viewsize.